quickstart_guides/rails.md

```diff
@@ -0,0 +1,85 @@
+address: support.aptible.com/quickstart/rails
+
+### Ruby on Rails Quickstart
+
+This guide assumes you have:   
+- An Aptible account,  
+- An SSH key associated with your Aptible user account, and  
+- The Aptible command line tool installed
+
+#### 1. Provision Your App  
+Tell the Aptible API that you want to provision an application. Until you push code and trigger a build, Aptible uses this as a placeholder.
+
+Use the `apps:create` command: `aptible apps:create $HANDLE`
+
+For example: 
+```
+aptible apps:create my-example-app
+```
+
+#### 2. Add a Dockerfile
+Aptible uses Docker to build and run applications. To tell Aptible how to build and run your specific application, you must include a Dockerfile in your repository.
+
+A few guidelines:  
+1. Name the file "Dockerfile" - one word, capital 'D', no extension  
+2. Place it in the root of your repository  
+3. Be sure to commit your Dockerfile to version control  
+
+Here is a sample Dockerfile for a Ruby on Rails app:
+```
+FROM quay.io/aptible/ruby:ruby-2.0.0
+
+RUN apt-get update && apt-get -y install libpq-dev nodejs
+ 
+ADD . /opt/rails
+WORKDIR /opt/rails
+RUN bundle install --without development test
+RUN bundle exec rake assets:precompile
+ 
+ENV PORT 3000
+EXPOSE 3000
+ 
+CMD bundle exec rails s -p $PORT
+```
+
+#### 3. Provision a Database
+By default, `aptible db:create $HANDLE` will provision a 10GB PostgreSQL database.
+
+`aptible db:create` will return a connection string on success. The host value is mapped to a private subnet within your stack and cannot be used to connect from the outside Internet. Your containerized app can connect, however.
+
+Add the connection string as an environmental variable to your app:
+```
+aptible config:add DATABASE_URL=$CONNECTION_STRING --app $APP_HANDLE
+```
+
+
+#### 4. Configure a Git Remote
+Add a Git remote named "aptible":
+```
+git remote add aptible git@beta.aptible.com:$APP_HANDLE.git
+```
+
+For example:
+```
+git remote add aptible git@beta.aptible.com:example-app.git
+```
+
+#### 5. Deploy Your App
+Push to the master branch of the Aptible git remote:
+```
+git push aptible master
```

Comment:
Looking very good!

We may need to make a very slight modification to the logged output for provisioning a VHOST, but it may be worth adding a small note to tell them how to visit their deployed app. For example:

> If your app is deployed successfully, near the end of the output from `git push`, you'll see the message:

> `VHOST example-app.on-aptible.com provisioned.`

> Visit [https://example-app.on-aptible.com](example-app.on-aptible.com) to rock and/or roll.